src/Uno.Extensions.Navigation.UI/RouteResolver.cs

```diff
@@ -7,6 +7,9 @@ public class RouteResolver : IRouteResolver
 
 	protected ILogger Logger { get; }
 
+	protected IRouteRegistry RouteMapss { get; }
```

Comment:
Should be `RouteMaps` instead of `RouteMapss` here right ?

Reply:
Corrected